src/OSPSuite.Core/Converter/v5_2/SerializationDimensionFactory.cs

```diff
@@ -27,6 +27,11 @@ public SerializationDimensionFactory(IDimensionFactory dimensionFactory, IDimens
          _allDimensions.AddRange(dimensionMapper.DummyDimensionsForConversion);
       }
 
+      public bool TryGetDimension(string dimensionName, out IDimension dimension)
```

Comment:
This whole class should go once we agree that we do not want to support such outdated projects